src/main/java/com/softech/coursesvc/CourseList.java

```diff
@@ -0,0 +1,16 @@
+package com.softech.coursesvc;
+
+import com.softech.coursesvc.entity.CourseEntity;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import java.util.List;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+
+public class CourseList {
+    List<CourseEntity> courses;
+    int noOfRecords;
```

Comment:
Make them private.. dont forget principal of encapsulation

Reply:
okay.

Reply:
Richa it is not letting me make this class private.
What should I do?